### DISCUSSION TITLE
Sign up for new account via MacOS Safari is buggy

### DISCUSSION THREAD

**Discussion by Niko102** [community/community#38446] [Other Feature Feedback, Questions, & Ideas]
Hello, I just signed up for a new GitHub account on MacOS Ventura Safari.  The sign up page doesn't work well in Safari.  At first, it would be excruciatingly slow watching one letter at a time pop up in the "Welcome" text before I could even begin signing up.  I had Safari full screen on a 4k monitor, when I made the Safari window much smaller, the page seemed to work a lot better.  May want to check the performance issues on Safari with the sign up form, or at least offer an "opt out" sign up form with out all the cute animations/graphics.

Thanks,
Niko

**Comment by ossanna16** [comment]
Hi there @Niko102 and welcome to our community! Thank you for reporting this!
